test/helpers.bash

```diff
@@ -223,7 +223,7 @@ function setup_test() {
 # Run crio using the binary specified by $CRIO_BINARY_PATH.
 # This must ONLY be run on engines created with `start_crio`.
 function crio() {
```

Comment:
as a follow up (or you can address it here) we should change this function name (crio-with-args?) it is not clear we're not running the crio binary in $PATH when calling it above.